bridge/contracts/libraries/StakingNFT/StakingNFT.sol

```diff
@@ -624,7 +624,7 @@ abstract contract StakingNFT is
         if (positionAccumulatorValue_ > state_.accumulator) {
             accumulatorDelta = type(uint168).max - positionAccumulatorValue_;
             accumulatorDelta += state_.accumulator;
-            positionAccumulatorValue_ = accumulatorDelta;
```

Comment:
After an overflow in one of the accumulators, the accumulator inside the position was set to a wrong value. E.g 
- accumulator overflow at 1000 and it's at 995
- position was created at 995
- accumulator was increased by 10. Now accumulator is 5

The old logic was yielding 
```
accumulatorDelta = 1000 - 995 = 5
accumulatorDelta += 5 = 10
positionAccumulatorValue_ = accumulatorDelta = 10
```
This way the `positionAccumulatorValue_` was being set to 10 while the global accumulator was in 5, this would lead to nasty errors and some tokens to be lost, because the user would only be able to collect profits when the global accumulator is >10. The correct value for `positionAccumulatorValue_` in this case (and always after a collect) is `state_.accumulator` (global accumulator)